src/JUDI4Flux.jl

```diff
@@ -116,4 +116,27 @@ module JUDI4Flux
             return EQT(d, m), Δ -> (nothing, grad_d(EQT, m, Δ), nothing)
         end
     end
+
+    # Fixed source forward modeling
+    struct Forward{T1}
+        F::T1   # forward modeling operator
+        q::judiVector
+    end
+
+    # Fixed source forward modeling: forward mode
+    function (FWD::Forward)(m::AbstractArray)
```

Comment:
This is acutally overkill with new JUDI since you can directly do `FWD.F(;m=m)` so you only need to add (could even be added to JUDI)

```
(FWD::Forward)(m::AbstractArray) = FWD.F(;m=m)* FWD.q
@adjoint (FWD::Forward)(m::AbstractArray) = FWD(m), Δ -> (nothing, reshape(transpose(judiJacobian(FWD.F(;m=m), FWD.q)) * vec(Δ), size(m)))
@adjoint *(F::judiModeling, x::judiVector) = *(F, x), Δ -> (nothing, transpose(F) * Δ)

```

Reply:
Thanks for your suggestion. I will adopt the first two lines. However,

why do we do 

`@adjoint *(F::judiModeling, x::judiVector) = *(F, x), Δ -> (nothing, transpose(F) * Δ)`

this does not seem necessary to me

Reply:
because by default would compute derivative w.r.t F tooo, same reason we explicitly define it fr J

Reply:
may wanna switch everything to chainrule at some point wold be clear btw